core/src/test/java/org/nextprot/api/core/service/statement/service/StatementAnnotationBuilderTest.java

```diff
@@ -0,0 +1,36 @@
+package org.nextprot.api.core.service.statement.service;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.nextprot.api.core.domain.annotation.AnnotationEvidence;
+import org.nextprot.api.core.domain.annotation.AnnotationEvidenceProperty;
+import org.nextprot.commons.statements.StatementBuilder;
+
+import org.junit.Assert;
+
+public class StatementAnnotationBuilderTest {
+
+	@Test
+	public void testBuildAnnotationEvidenceKey() {
```

Comment:
Here you have to test the generated key, So you have to instantiate statementannotationbuilder and pass the annotationevidence there and test for the generated key. 

StatementAnnotationBuilder = new StatmentAnnotationbuilder()
Here pass some null values for the services to be passed. Then you can call buildAnnotationEvidenceKey() and test the result